external/fv3fit/fv3fit/_shared/models.py

```diff
@@ -305,12 +324,26 @@ def load(cls, path: str) -> "OutOfSampleModel":
 
         base_model = io.load(config["base_model_path"])
         novelty_detector = io.load(config["novelty_detector_path"])
+        cutoff = 0
         if "cutoff" in config:
             cutoff = config["cutoff"]
-        else:
-            cutoff = None
 
         assert isinstance(novelty_detector, NoveltyDetector)
 
-        model = cls(base_model, novelty_detector, cutoff)
+        default_tapering_config = {
```

Comment:
Can these default just be handled in the definition of the individual tapering functions and removed from the OOS model?

Reply:
That should work, as long as it's fine that `cutoff` gets passed to `get_taper_function`.